hs-ucan/library/Web/UCAN/Types.hs

```diff
@@ -58,30 +65,49 @@ import qualified Web.UCAN.Internal.UTF8                        as UTF8
 
 -- Reexports
 
+import           Web.UCAN.Capabilities.Types
+import           Web.UCAN.Claims.Types
+import           Web.UCAN.Header.Types
+import           Web.UCAN.Nonce.Types
+import           Web.UCAN.Proof.Types
 import           Web.UCAN.RawContent
 
+
 -- | An RFC 7519 extended with support for Ed25519 keys,
 --     and some specifics (claims, etc) for Fission's use case
-data UCAN fct rsc ptc = UCAN
-  { header :: Header
-  , claims :: Claims fct rsc ptc
-  , sig    :: Signature.Signature
+data UCAN fct res abl = UCAN
+  { header     :: Header
+  , claims     :: Claims fct res abl
+  , signedData :: RawContent
```

Comment:
I'm storing the `signedData` inside the UCAN.
The downside of this is that the UCAN type can represent weird stuff, e.g. the signedData might not match what's in `claims`.
However, it turns out that we've basically had this before, where lots of functions would have taken both `RawContent` and `UCAN` as an argument or return both `RawContent` as well as `UCAN`.
I just noticed the cohesion and made it more coherent.